drf_multiple_model/mixins.py

```diff
@@ -172,14 +193,32 @@ def queryList_sort(self, results):
 
 class Query(object):
 
-    def __init__(self, queryset, serializer, label=None, filter_fn=None, ):
+    def __init__(
+        self,
+        queryset,
+        serializer,
+        label=None,
+        filter_fn=None,
+        serializer_kwargs=None
+    ):
         self.queryset = queryset
         self.serializer = serializer
         self.filter_fn = filter_fn
         self.label = label
+        self.serializer_kwargs = serializer_kwargs if serializer_kwargs else {}
```

Comment:
I think the best place to set this as an empty dictionary should be before [line 106](https://github.com/startupgrind/DjangoRestMultipleModels/pull/1/files#diff-70a9e7448d069fc7948ab5283daca800R106) before using the kwargs in the serialized.